### DISCUSSION TITLE
Extension activation failed: "No access to GitHub Copilot found. You are currently logged in as (another account of mine)."

### DISCUSSION THREAD

**Discussion by 0danteh** [community/community#79215] [Copilot Conversations]
### Select Topic Area

Question

### Body

I have subscribed to github copilot today, but the extension is recognising another account of mine (rather the one that I have used for my github subscription). How can I solve this?

**Comment by ayim** [comment]
@dante-tech which IDE are you in?

**Comment by 0danteh** [comment]
@ayim I was on VSCode, however I managed to solve the problem! Thank you for your interest, I wish you a beautiful day!

**Reply by h1ghs34** [reply]
how? Let us know.
